src/tools/rust-analyzer/crates/proc-macro-api/src/version.rs

```diff
@@ -120,13 +120,18 @@ pub fn read_version(dylib_path: &AbsPath) -> io::Result<String> {
     let version = u32::from_be_bytes([dot_rustc[4], dot_rustc[5], dot_rustc[6], dot_rustc[7]]);
     // Last supported version is:
     // https://github.com/rust-lang/rust/commit/0696e79f2740ad89309269b460579e548a5cd632
```

Comment:
It's so outdated that when I saw this I wasn't sure if it's supposed to point at the oldest or the newest version that's supported.

Reply:
I think it's supposed to point at the newest, because
1. > Last
2. This points to 6, but the oldest supported version is 5, so I don't think it could have been added as an oldest one